components/emoji_picker/emoji_picker_popover.story.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <dt-popover
+    :open="emojiPickerOpened"
+    initial-focus-element="#searchInput"
+    padding="none"
+    @opened="(open) => { emojiPickerOpened = open }"
+  >
+    <template #anchor>
+      <dt-button
+        size="sm"
+        circle
+        importance="clear"
+        @click="toggleEmojiPicker"
+      >
+        <template #icon>
+          <dt-icon
+            name="satisfied"
+            size="300"
+          />
+        </template>
+      </dt-button>
+    </template>
+    <template #content>
+      <dt-emoji-picker
+        ref="emojiPickerRef"
+        :skin-tone="isSkinTone"
+        :skin-selector-button-tooltip-label="$attrs.skinSelectorButtonTooltipLabel"
+        :tab-set-labels="$attrs.tabSetLabels"
+        :recently-used-emojis="$attrs.recentlyUsedEmojis"
+        :search-results-label="$attrs.searchResultsLabel"
+        :search-no-results-label="$attrs.searchNoResultsLabel"
+        :search-placeholder-label="$attrs.searchPlaceholderLabel"
+        @skin-tone="isSkinTone = $event"
+        @selected-emoji="$attrs.selectedEmoji"
+      />
+    </template>
+  </dt-popover>
+</template>
+
+<script>
+import DtEmojiPicker from './emoji_picker.vue';
+import DtButton from '@/components/button/button.vue';
+import DtIcon from '@/components/icon/icon.vue';
+import DtPopover from '@/components/popover/popover.vue';
+
+export default {
+  name: 'DtEmojiPickerDefaultTemplate',
+
+  components: {
+    DtEmojiPicker,
+    DtButton,
+    DtIcon,
+    DtPopover,
+  },
+
+  data () {
+    return {
+      isSkinTone: this.$attrs.skinTone,
+      emojiPickerOpened: true,
```

Comment:
change this to only be opened by default on visual tests please

Reply:
hmm I added the percy prop, but seems not to work... I replicated the `datepicker.stories.js` and `dropdown.stories.js` without success... 🤔